_data/workinggroups.yaml

```diff
@@ -0,0 +1,21 @@
+- name: CI Working Group
+  url: meetings/CI-workinggroup
```

Comment:
CI WG has its own repo, meetings URL at:

https://github.com/AcademySoftwareFoundation/wg-ci/tree/main/meetings

Reply:
Thanks! Did all the old meeting notes get migrated to the new site?